# SingleCellGRN
Inferring Gene Regulatory Networks from single cell data with perturbations

This project aims to leverage recent advancements in single-cell perturbation-based sequencing to enhance our understanding of gene regulatory networks (GRNIs). While previous studies (Secilmis et al., 2022) have demonstrated the effectiveness of GRNI methods based on known perturbations, such experiments have not yet been conducted at a large scale with single cells, despite the availability of the technology. However, a recent breakthrough by Replogle et al. (2022) presents a large-scale perturbation-based single-cell dataset comprising 8249 genes perturbed across 2.5 million cells. This dataset offers immense potential for preprocessing in various ways to address the inherent incompleteness in each cell's transcriptome. Our project will initially focus on utilizing the bulked data format, where single cells with the same perturbation are grouped together. This approach not only facilitates comparison with raw data from the ENCODE project (van Nostrand et al., 2020), which was conducted in bulk for the same K562 cell line, but also provides a foundation for exploring and optimizing preprocessing strategies to extract meaningful insights from single-cell perturbation data.
